Xamarin.Forms.Platform.Android/TapGestureHandler.cs

```diff
@@ -39,7 +39,7 @@ public bool OnTap(int count, Point point)
 			if (children != null)
 				foreach (var recognizer in children.GetChildGesturesFor<TapGestureRecognizer>(recognizer => recognizer.NumberOfTapsRequired == count))
 				{
-					recognizer.SendTapped(view);
+					recognizer.SendTapped(view, point);
```

Comment:
Here need add `pixelTranslation` function. For example, look like made in:
https://github.com/xamarin/Xamarin.Forms/blob/d150e22238ea184a2f9aa3c4b7dcfb134db6f51d/Xamarin.Forms.Platform.Android/PanGestureHandler.cs#L27-L32

Also below.